config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :users, only: [:index, :show]
+  resources :posts, only: [:index, :show]
```

Comment:
- [ ] Nice adding routes for your app however what you did it's not as required right now it create sperate routes for both of them but the routes should be as in the picture in the requirements so please try  to modify your code in order to make the same routes as in the requirements
- Required routes 
![image](https://user-images.githubusercontent.com/61790587/193641513-65b03c66-427f-43e2-9806-7265d260bee8.png)
- Yours 
![Uploading Screen Shot 2022-10-03 at 18.38.53.png…]()

Reply:
Hi @youmari .

When I run rails routes, I get exactly the required routes, as you can see in my terminal's screenshot:
![image](https://user-images.githubusercontent.com/100212928/193645695-00bed98e-95a1-4828-9f13-c59ac6e5c3a4.png)

So, I already have what you are asking for. Or I don't understand what you mean. Please, can you explain it again or tell me what exactly should I code as routes ?

By the way, I have reviewed 4 different repos approved last week with the same code as I did. So, I hope you understand why I'm confused by your required changes.

Thank you.

Reply:
@DarioAlessioR That's what I mean in order to get the required routes you've should nest the posts routes inside the user routes in order to get this result but at the first you wasn't doing that